Fundamentals/Data/Relations/index.md

```diff
@@ -0,0 +1,103 @@
+
+Right click the 'Relation Types' folder to create your new Relation Type
+
+![Create Relation Type](images/create-relation-type.png)
+
+Provide the Relation Type with a name (this will generate it's alias).
+
+Choose the 'direction' of the relationship, eg usually Bidirectional to get the benefits of relations.
+
+Define the type of one object in the relation (confusingly this is called 'Parent' but that only defines which column in the database this value is stored in, for a bidirectional relationship it doesn't matter which type of entity is defined as the parent or the child, but if there is a natural 'one thing' will be related to lots of 'other things', then choose that thing as the parent. (really they should be Uncles/Aunts and Nephews/Nieces - but there isn't a common non-gendered term for this family relationship, although Piblings and Niblings has started to become popular).
+
+Choose the different types for each entity in the relationship.
+
+Currently the available types are:
+
+Document(Content), Media, Member, Document Type, Media Type, Member Type, Member Group, Data Type, Root, Recycle Bin
+
+eg. for relating Members to their uploaded Images, we might create a 'Member Images' relation
+
+![Member Images](images/member-images.png)
+
+'Is Dependency' - in the functionality to prevent an item from being deleted if it's in use, some performance gains have been found when querying if a media item is a dependency - by adding an 'Is Dependency' field to the relationship, generally for your custom relations, you'll set this to false.
+
+Click Create, and you'll see your new Relation Type created in the Relation Types tree, and be able to see the all important 'Alias' that you'll need to make note of when working with Relations.
+
+![Member Images](images/relation-alias.png)
+
+## Viewing Relations
+
+If you examine one of the existing Relation Types, you will see the option to view Relations - this is just a long list of all the objects that have been related for this specific Relation Type, there is no backoffice functionality here to create, edit or delete a Relation.
+
+![View Relations](images/relation-alias.png)
+
+
+## Creating Relations
+
+You can create Relations using the RelationService API via code.
+
+[Some examples are provided here in the RelationService Documentation Page](../../documentation/reference/management/services/relationservice/)
+
+### Use cases
+Commonly you might create a 'Relation' between two objects either as 
+
+a response to a backoffice event, eg a content item being published that has picked several other content items - you might add a relationship between these items, to make querying between them easier. 
+
+Or 
+
+in the front end of your site, a logged in member might upload images to your site, and you create them programatically in the Media Section and create a Relation between the Member and the Media Item, so you can retrieve the Members images to display in a gallery.
```

Comment:
```suggestion
- in the front end of your site. A logged-in member might upload images to your site. You can create them programmatically in the Media Section and create a Relation between the Member and the Media Item, so you can retrieve the Members images to display in a gallery.
```

Reply:
hehe, apologies this is nonsense :-P how about

A logged-in member on the front end of your website might have the facility to upload images. In response, you could store the photos programmatically in the Media Section and at the same time, create a Relation to record the relationship between the member and their pictures. On an image gallery page, you could then display all the gallery images for the current logged-in Member.

Reply:
Looks good 💪

Reply:
@eshanrnh tweaked and pushed!